io_scene_niftools/modules/nif_import/armature/__init__.py

```diff
@@ -319,6 +319,9 @@ def check_for_skin(self, n_root):
             NifLog.debug(f"{n_block.name} has skinning.")
             # one is enough to require an armature, so stop
             return
+        # force import of nodes as bones, even if no geometries are present
+        if NifOp.props.process == "SKELETON_ONLY":
```

Comment:
Probably want to pull this out to a constant / enum as used in a few places in the code.